### PR TITLE
New Synthetic vendor sprites!

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Title.

## Why It's Good For The Game

Did you know that the req Operational vendor has the same sprite as synthetic? It's time to change that.

Thanks to CM SS13 for sprites.

## Changelog
:cl:
imageadd: New Synthetic vendor sprites
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
https://user-images.githubusercontent.com/6610922/149997516-a2cb9a25-d96c-4692-8247-b986953b46bb.mp4

**Comment by Losenis** [comment]
>Snowflake

I feel attacked

**Comment by Kuro020** [comment]
> 2022-01-18.10-14-52.mp4

video isnt working 4head, post pics

**Review by Kuro020 — APPROVED** [review]

**Comment by Losenis** [comment]
image

**Comment by novaepee** [comment]
> > 2022-01-18.10-14-52.mp4
> 
> video isnt working 4head, post pics

nani, it should be working

**Comment by novaepee** [comment]
image

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 1]
```diff
@@ -479,9 +479,9 @@
 /obj/machinery/marine_selector/clothes/synth
 	name = "M57 Synthetic Equipment Vendor"
 	desc = "An automated synthetic equipment vendor hooked up to a modest storage unit."
-	icon_state = "synth"
-	icon_vend = "synth-vend"
-	icon_deny = "synth-deny"
+	icon_state = "snowflake"
```
Comment (BraveMole) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:482]: why funny name

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 1 above)
Comment (novaepee) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:482]: Req's opertional vendor has synth. We would have to change req's eperational vendor icon path if we want synth vendor to have name.

**Pull request merged**